site/app/_components/PlayerHero.tsx

```diff
@@ -43,13 +43,22 @@ export default function PlayerHero(props: PlayerHeroProps) {
     setWidth(parseInt(event.currentTarget.value, 10));
   };
 
+  const isPortraitAspectRatio = (ratio: string) => {
+    const match = ratio.match(/^(\d+)\s*\/\s*(\d+)$/);
+    if (!match) return false;
+
+    const width = parseFloat(match[1]);
+    const height = parseFloat(match[2]);
+    return height > width;
+  };
```

Comment:
why use the string notation and add more logic to parse it? this feels more brittle and prone to errors in the future.